test/fixtures/drafts.json

```diff
@@ -8,7 +8,8 @@
     "ReplyTo": "myemail@example.com",
     "DateCreated": "2010-08-19 16:08:00",
     "PreviewURL": "http://createsend.com/t/r-E97A7BB2E6983DA1",
-    "PreviewTextURL": "http://createsend.com/t/r-E97A7BB2E6983DA1/t"
+    "PreviewTextURL": "http://createsend.com/t/r-E97A7BB2E6983DA1/t",
+    "Tags": ["Tags5"]
```

Comment:
To test the 'Draft' in CampaignSummary

Reply:
might be good to specify https links here